### PR TITLE
Add k3s-tests subchart in serverless chart

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

needed for https://github.com/kyma-project/test-infra/pull/2939

Changes proposed in this pull request:

- introduce special subchart, that will be only installed during k3s pipeline

**Related issue(s)**

#10001

### CODE REVIEW

File: tests/function-controller/cmd/main.go [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+	"github.com/kyma-project/kyma/tests/function-controller/pkg/step"
+	"github.com/kyma-project/kyma/tests/function-controller/testsuite"
+	"github.com/kyma-project/kyma/tests/function-controller/testsuite/scenarios"
 
 	controllerruntime "sigs.k8s.io/controller-runtime"
 )
 
 var availableScenarios = map[string][]testSuite{
-	"serverless-integration": {scenarios.SimpleFunctionTest},
+	"serverless-integration": {scenarios.SimpleFunctionTest, scenarios.GitopsSteps},
```
Comment (aerfio) [inline tests/function-controller/cmd/main.go:21]: `gitopssteps` does not work due to the fact that we create DestinationRule there

**Review by m00g3n — APPROVED** [review]

File: resources/serverless/charts/k3s-tests/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# k3s-tests
+
+## Overview
+
+This project contains the chart with test Job for function-controller.
+
+## Details
+
+This chart _SHOULD NOT_ be installed with Kyma.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply resources/serverless/charts/k3s-tests/README.md:9]: I would also mention `why` ;)

File: resources/serverless/charts/k3s-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# k3s-tests
+
+## Overview
+
+This project contains the chart with test Job for function-controller.
```
Comment (kazydek) [inline resources/serverless/charts/k3s-tests/README.md:5]:
```suggestion
This project contains a chart with the test Job for the Function Controller.
```

Comment (kazydek) [reply resources/serverless/charts/k3s-tests/README.md:5]: Maybe you can mention in one sentence what this Job does?

**Review by kazydek — APPROVED** [review]

**Review by m00g3n — APPROVED** [review]

**Pull request merged**